src/components/DialogCard.vue

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+import { defineComponent, computed } from 'vue';
+
+// Component used to render dialog window which you can fill with custom content
```

Comment:
This multi-line comment (for multi-line commenting use `/* */` please) should be as component doc header on the first line.

Reply:
Updated component doc header.